source/common/quic/envoy_quic_proof_verifier.cc

```diff
@@ -47,7 +48,8 @@ class QuicValidateResultCallback : public Ssl::ValidateResultCallback {
     std::string error;
     if (!succeeded) {
       error = error_details;
-    } else {
+    } else if (!accept_untrusted_ || !Runtime::runtimeFeatureEnabled(
```

Comment:
Why do we need this for E-M test to pass?

Reply:
Oh, I see. The test uses IP address as the host but the test server sends a test cert for lyft.com. Fair enough!